lib/build.rb

```diff
@@ -66,6 +66,13 @@ def query_branches
       end
       $logger.debug("Querying potential build: #{b.name}")
       branch_details = github_query(@client) { @client.branch(@repository, b.name) }
+      skip_message_present = false
+      begin
+        skip_message_present = branch_details.commit.message['[decent_ci_skip]']
+      rescue
+        # Ignored
+      end
+      next if skip_message_present and branch_details.name != 'develop'  # only skip if we have the msg on a non-develop branch
```

Comment:
Only skip if we do have the skip message and we also are not on the develop branch.